projects/media-viewer/src/lib/annotations/comment-set/comment-set.component.html

```diff
@@ -1,4 +1,11 @@
-<div #container class="split right comment-container" [style.height.px]="height" (click)="onContainerClick($event)">
+<div #container
+     [ngClass]="{ 'right comment-container': true, 'show-comments-panel': showCommentsPanel }"
+     [style.height.px]="height" (click)="onContainerClick($event)">
+  <button [id]="showCommentsPanel ? 'collapseCommentsPanel' : 'expandCommentsPanel'"
```

Comment:
why is the id changing? maybe better to have it as a class?

Reply:
agreed, changed